### ISSUE TITLE
Not working outside the home

### ISSUE THREAD

**Issue by keithah** [KhaosT/homebridge-nest-cam#25]
This plugin works great when I'm connected to my home wifi. It takes about 5 seconds for 'Live' on my Synology NAS, which is pretty good.

However, when I run it disconnected from Wifi, I immediately get a 'not responding' when I click on the video. I am running this in Docker and I'm wondering if I need to open up any ports? I have a Apple TV 4 as a bridge.

Thanks!

**Comment by DCMarine** [comment]
Is the “not responding” happening only on your video streams?   There should be no reason to open any ports to get this work ok HomeKit.   I’ve seen issues with Plex running in docker, inside network it is perfect and outside the network it was bad.  Had to tune resources for Docker to get it work right.    Just a thought.

**Comment by keithah** [comment]
I don't see Camera as an option:
![image](https://user-images.githubusercontent.com/684920/38515907-8666bd04-3bea-11e8-96e1-b41c450d1e2b.png)

Am I missing something?

@DCMarine Yes, only with video streams. I'm using Host mode, so shouldn't have any network issues..

**Comment by seannymurrs** [comment]
I appear to be having the same issue. Were you ever able to figure out he cause of the issue and how to fix it?

**Comment by keithah** [comment]
Nope, unfortunately.

**Comment by shereefalkady** [comment]
Yeah- I’m using SecuritySpy on Mac.... works great on WiFi — and still images update on LTE... but video stream gives not responding

**Comment by adamsiem** [comment]
Will the [new HomeKit Secure Video functionality](https://9to5mac.com/2019/10/04/early-look-at-the-ui-for-apples-homekit-secure-video/) be An alternative to resolve this issue? 

Are the Nest Cameras going to be able to work with this functionality? It’s not clear if the Home Hub/Homebridge can do the encryption, or if the camera itself needs to support executing the encryption.

Sidenote: It appears this only supports up to 5 cameras. I have 6 Nest Cameras which I’m trying to bring onto HomeKit with remote viewing.
